external-import/ransomwarelive/src/lib/ransomConn.py

```diff
@@ -373,35 +373,41 @@ def stix_object_generator(self, item, group_data):
         resolved_ip = None
         # Creating Victim object
         if len(item.get("post_title")) > 2:
```

Comment:
Suggestion to reduce duplication and remove unused argument

```suggestion
        post_title=item.get("post_title", "")
        victim_name, identity_class = (post_item, 'organization') if len(post_title > 2) else  (post_item + ":<)", 'individual')
        victim = Identity(
                id=pycti.Identity.generate_id(victim_name, identity_class ),
                name=victim_name,
                identity_class=identity_class ,
                created_by_ref=self.author.get("id"),
                object_marking_refs=[self.marking.get("id")],
            )
```